mmcv/ops/csrc/common/pytorch_cpp_helper.hpp

```diff
@@ -1,6 +1,6 @@
 #ifndef PYTORCH_CPP_HELPER
 #define PYTORCH_CPP_HELPER
-#include <torch/extension.h>
+#include <torch/types.h>
```

Comment:
Do we have to modify this?

Reply:
Yes. If not, the error `error: invalid static_cast from type ‘const torch::OrderedDict<std::basic_string<char>, at::Tensor>’ to type ‘torch::OrderedDict<std::basic_string<char>, at::Tensor>&’` cannot be handled.